hmpps-prisoner-search-indexer/src/test/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/indexer/services/events/HmppsDomainEventsEmitterIntTest.kt

```diff
@@ -196,7 +196,7 @@ class HmppsDomainEventsEmitterIntTest : IntegrationTestBase() {
 
   @Test
   fun `e2e - will send prisoner updated event to the domain topic`() {
```

Comment:
all these tests were seeding a booking with a null bookingId that made no sense and caused the receive domain event to fire accidentally